docs/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -43,13 +43,15 @@ For the sake of brevity, this guidance shows the most structured way to use `Htt
 
 ## How to use Typed Clients with HttpClientFactory
 
-The following diagram shows how Typed Clients are used with HttpClientFactory.
+So, what's a "typed client" anyway? It's just an `HttpClient` that's configured upon injection by  the `DefaultHttpClientFactory`.
 
-![Diagram with an MVC controller using an injected ClientService, which internally is using a configured HttpClient by HttpClientFactory and Polly's policies](./media/image3.5.png)
+The following diagram shows how Typed Clients are used with `HttpClientFactory`.
```

Comment:
do you need to capitalize typed clients? end with :

Reply:
It's used like that here: https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests, so I guess it's like a proper name.